master/buildbot/reporters/gerrit.py

```diff
@@ -300,6 +300,9 @@ def buildComplete(self, key, build):
             result = _handleLegacyResult(result)
             self.sendCodeReviews(build, result)
 
+    def checkConfig(self, *args, **kwargs):
```

Comment:
Looks like I did not realize the problem at first glance.
You say the it did broke in .0.9.7. Do you have an idea of what did create the issue?
would you like to git bisect?